app/Http/Controllers/API/v1/LikesController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Http\Controllers\API\v1;
+
+use App\Exceptions\StatusAlreadyLikedException;
+use App\Http\Controllers\API\ResponseController;
+use App\Http\Controllers\Controller;
+use App\Http\Controllers\StatusController as StatusBackend;
+use App\Http\Resources\UserResource;
+use App\Models\Like;
+use App\Models\Status;
+use App\Models\User;
+use http\Env\Response;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Resources\Json\AnonymousResourceCollection;
+use Illuminate\Support\Facades\Auth;
+
+class LikesController extends ResponseController
+{
+    /**
+     * @param int $status
+     * @return AnonymousResourceCollection
+     * @todo maybe put this in separate controller?
+     */
+    public function show(int $status): AnonymousResourceCollection {
+        return UserResource::collection(
+            User::whereIn('id', Like::where('status_id', $status)->select('user_id')->get())->get()
```

Comment:
This should work withour this get(). Can save one query. Please check before commit.
```suggestion
            User::whereIn('id', Like::where('status_id', $status)->select('user_id'))->get()
```